README.md

```diff
@@ -1,6 +1,6 @@
 # HBStampieBundle
 
-[![Build Status](https://secure.travis-ci.org/henrikbjorn/HBStampieBundle.png)](http://travis-ci.org/henrikbjorn/HBStampieBundle)
+[![Build Status](https://secure.travis-ci.org/Stampie/HBStampieBundle.png)](http://travis-ci.org/Stampie/HBStampieBundle)
```

Comment:
you should update the URL to the uptodate  one (SVG, no `secure` subdomain, and link going to HTTPS directly)